src/engine/DnCManager.h

```diff
@@ -51,7 +51,7 @@ class DnCManager
     /*
       Perform the Divide-and-conquer solving
     */
-    void solve( unsigned timeoutInSeconds );
+    void solve( unsigned timeoutInSeconds, bool restoreTreeStates=true );
```

Comment:
perhaps, it is better to be aligned with the default option for restoreTresStates.

So, here it should be `bool restoreTreeStates=false`